job_spec/RIVER_WIDTH.yml

```
@@ -41,9 +41,21 @@ RIVER_WIDTH:
         - --resolution
         - 10.0
         - Ref::granules
-      timeout: 36000
-      vcpu: 1
-      memory: 126000
+      timeout: TODO
+      vcpu: TODO
+      memory: TODO
```

```suggestion
      timeout: 36000
      vcpu: 1
      memory: 31600
```

For RTC we can just use the values from the RTC job spec